src/m365/entra/commands/m365group/m365group-add.ts

```diff
@@ -16,8 +16,8 @@ interface CommandArgs {
 
 interface Options extends GlobalOptions {
```

Comment:
Not really part of the issue, but important for the command. Could you also add an #initTypes function to express the type of every option, please? The boolean values are already typed but it's also important to type string options.